src/universal/mutations/CreateReflectionMutation.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Creates a reflection for the retrospective meeting.
+ *
+ * @flow
+ */
+import {commitMutation} from 'react-relay';
+import {Environment, RecordProxy, RecordSourceSelectorProxy} from 'relay-runtime';
+
+import clientTempId from 'universal/utils/relay/clientTempId';
+import makeEmptyStr from 'universal/utils/draftjs/makeEmptyStr';
+
+type Variables = {
+  content?: string,
+  meetingId: string,
+  retroPhaseItemId: string,
+  sortOrder: number
+};
+
+type CompletedHandler = (response: ?Object, errors: ?Array<Error>) => void;
+
+type ErrorHandler = (error: Error) => void;
+
+graphql`
+  fragment CreateReflectionMutation_team on CreateReflectionPayload {
+    meeting {
+      id
+    }
+    reflection {
+      content
+      id
+      isViewerCreator
+      retroPhaseItemId
+      sortOrder
+    }
+}
+`;
+
+const mutation = graphql`
+  mutation CreateReflectionMutation($content: String, $meetingId: ID!, $retroPhaseItemId: ID!, $sortOrder: Float!) {
```

Comment:
i'm surprised this didn't throw an error since retroPhaseItemId is marked as required here & not in the server mutation.

After looking it over, I'm a big dumb idiot. I should make the retroPhaseItemId required on the server & remove meetingId, since I can get to the meetingId from the retrophaseitemid. i'll change  that in a PR to follow